hf

```diff
@@ -54,6 +54,7 @@ def main
   options[:quiet] = flags.include?('--quiet') || flags.include?('--q')
   options[:yes] = flags.include?('--yes') || flags.include?('--y')
   options[:echo] = flags.include?('--echo')
+  options[:skip_staging] = flags.include?('--dry')
```

Comment:
#2 think on option name. Examples: --na, --no-add, --skip-add, --sa, etc.

Reply:
Probably --skip-add option name is more readable